pyseir/inference/infer_rt.py

```diff
@@ -195,6 +204,9 @@ def apply_gaussian_smoothing(self, timeseries_type, plot=False):
             idx_start = 0
```

Comment:
So, I think this line is now redundant, as an input array that is all zeros would not pass the later max threshold requirement. I think it would be good to remove this, to make the code more clear. What do you think?

Reply:
For v2, we could override values below the threshold to NaN, or back in infer_rt.py after bayesian updating and time lag.